includes/class-wc-ecfb-front-end.php

```diff
@@ -236,6 +242,7 @@ public function checkout_billing_fields( $fields ) {
 		if ( isset( $fields['billing_postcode'] ) ) {
 			$new_fields['billing_postcode'] = $fields['billing_postcode'];
 			$new_fields['billing_postcode']['class'] = array( 'form-row-last', 'address-field' );
+			$new_fields['billing_postcode']['type'] = 'tel';
```

Comment:
Se usar isso com o CEP vai deixar de funcionar corretamente para quem vende dentro e fora do Brasil, o plugin suporta uma opção para escolher os campos de CPF/CNPJ se não for venda no Brasil, e ai tem vários lugares que usam letras no `postcode`, exemplos: Canadá e Inglaterra.